R/RequiredCols.R

```diff
@@ -0,0 +1,231 @@
+# Lists can be used within TADA_OrderCols, TADA_GetTemplate, TADA_CheckRequiredFields, TADA_AutoFilter, TADA_RetainRequired
+
+# ordered list of TADA workflow required columns to be retained in data frame
+require.cols <- c(
```

Comment:
Would it make sense to store the column names in a csv file? An additional column could store which "group" they are in (Result Value, Detection Limits, etc.)? Others could specify which are generated and which are required.

It might be useful to be able to easily specify subsets of the whole list in current or future functions.